Firebend.AutoCrud.Core/Implementations/Caching/JsonEntityCacheSerializer.cs

```diff
@@ -0,0 +1,32 @@
+#nullable enable
+using Firebend.AutoCrud.Core.Interfaces.Caching;
+using Newtonsoft.Json;
+
+namespace Firebend.AutoCrud.Core.Implementations.Caching;
+
+public class JsonEntityCacheSerializer : IEntityCacheSerializer
+{
+    public string? Serialize<T>(T value) where T : class
+    {
+        try
+        {
+            return JsonConvert.SerializeObject(value);
+        }
+        catch
+        {
+            return null;
```

Comment:
can we add a logger here so we can log errors?

Reply:
I'll move the serializer out of the cache options and inject it in the service so that we can inject a logger